collect_app/src/main/java/org/odk/collect/android/geo/MbtilesFile.java

```diff
@@ -204,6 +201,11 @@ private static String detectContentType(File file) throws MbtilesException {
         }
     }
 
+    private static SQLiteDatabase openSqliteReadOnly(File file) {
+        return SQLiteDatabase.openDatabase(
+            file.getPath(), null, OPEN_READONLY | NO_LOCALIZED_COLLATORS);
```

Comment:
What are the implications of the `NO_LOCALIZED_COLLATORS` flag? [The docs](https://developer.android.com/reference/android/database/sqlite/SQLiteDatabase#NO_LOCALIZED_COLLATORS) say "You must be consistent when using this flag to use the setting the database was created with."

Reply:
I copied this from the existing MBTiles classes.

The LOCALIZED collator is an Android-specific feature that controls the other in which query results are sorted.  Since we don't ask SQLite to do any sorting, we don't need it.  I assume the other MBTile classes do this to save work.

Reply:
Thanks!

Reply:
The consequence of disabling the LOCALIZED collator is that it won't be able to open database files that have the LOCALIZED order set on a column.  But MBTiles files aren't created on Android devices, so I don't think this can happen.